python/tank/descriptor/descriptor.py

```diff
@@ -339,8 +339,15 @@ def version(self):
 
     def get_path(self):
         """
-        Returns the path to the folder where this item either currently resides
-        or would reside in case it existed locally.
+        Returns the path to a location where this item is cached.
+
+        When locating the item, any bundle cache fallback paths
```

Comment:
Really? I just checked the code and I wasn't aware of that. I always assumed fallback bundle cache paths meant  "if it's not in my main one then look for these other paths".

Should we have some performance considerations here then? I'm assuming the local bundle cache is faster to scan for files and should probably be the first? I'm curious what you think.

Reply:
maybe? we can discuss this further if you like, but seems out of scope of this PR. I was merely updating the docstring.

Reply:
Totally agree. Let's not do this now.